lnrpc/rpc.proto

```diff
@@ -1169,6 +1169,12 @@ message Channel {
     The minimum satoshis the other node is required to reserve in its balance.
     */
     int64 remote_chan_reserve_sat = 21 [json_name = "remote_chan_reserve_sat"];
+
+    /**
+    The percentage of the channel's lifetime that the remote peer has been
+    online.
+    */
+    int64 uptime_percentage = 22 [json_name = "uptime_percentage"];
```

Comment:
Maybe we can also add the channel life time here, as we have it available anyway. And I think we should also mark the fields as 'experimental' in the comment, so that we can make breaking changes later if we want to.

Reply:
Sounds good, it also makes it less likely that the uptime is misinterpreted by users as applying to the whole lifetime of the channel. 

How about:
`observed_lifetime`: total seconds we have been monitoring the channel
`observed_uptime`: total seconds we have observed the peer online during monitoring

Then users can do their own calculations, and there's no rounding to be dealt with. There's probably a better var name than `observed` because that makes is sound like we didn't know about the channel before that, but I'll scour synonyms.com later :)

Reply:
Sounds good. Sounds better. I don't think the `observed` prefix is needed, is it?

Reply:
How should I mark fields as experimental? I can't see any examples in the code. 
Thinking of something along these lines in the field comments?
```
    [experimental]: this field is experimental and may be subject to change.
```

Reply:
We indeed don't have that yet. All our experimental fields are in sub-servers atm. Yes, something along those lines.

Reply:
IMO we don't have to mark them experimental. We can rather deprecate and add new fields later if we need to.